.gitignore

```diff
@@ -7,3 +7,5 @@ _version.py
 venv/
 tmp/
 .vscode/
+build/
+.DS_Store
```

Comment:
Why ?

Reply:
Yes, those files are noise.
I'm adding them to the ignore so that no one accidentally pushes them. 

Any concerns?

Reply:
Nope! Just to make sure its not by accident